config/config.go

```diff
@@ -1,35 +1,66 @@
 package config
 
 import (
-	"fmt"
 	"log"
-	"path/filepath"
-
-	"github.com/spf13/viper"
+	"os"
 )
 
-// Init is an exported method that takes the environment starts the viper
-// (external lib) and returns the configuration struct.
-func Init(cfgFile string) {
-	if cfgFile != "" { // enable ability to specify config file via flag
-		viper.SetConfigFile(cfgFile)
-	} else {
-		viper.SetConfigName(".taco") // name of config file (without extension)
-		viper.AddConfigPath("$HOME") // adding home directory as first search path
+type Config struct {
+	AWS_Region          string
+	Dynamo_Db           string
+	Dynamo_Disable_SSL  bool
+	Resource_Table_Name string
+}
+
+func NewConfig() *Config {
+	return &Config{
+		AWS_Region:          aws_region(),
+		Dynamo_Db:           dynamo_db(),
+		Dynamo_Disable_SSL:  dynamo_disable_ssl(),
+		Resource_Table_Name: resource_table_name(),
+	}
+}
+
+func aws_region() string {
+	var region string
+	region = os.Getenv("AWS_REGION")
+	if region == "" {
+		region = "localstack"
+		log.Printf("AWS_REGION: Using default [localstack].")
+	}
+	log.Printf("AWS_REGION: Found setting [%s]", region)
+	return region
+}
+
+func dynamo_db() string {
```

Comment:
Just one more suggestion. Can we name this `DYNAMO_DB_ENDPOINT/dynamo_db_endpoint`? that would follow the AWS terminology

Reply:
Sure.